From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 18 Mar 2017 20:47:58 +0000
Subject: fjes: Disable auto-loading
Bug-Debian: https://bugs.debian.org/853976
Forwarded: no

fjes matches a generic ACPI device ID, and relies on its probe
function to distinguish whether that really corresponds to a supported
device.  Very few system will need the driver and it wastes memory on
all the other systems where the same device ID appears, so disable
auto-loading.

---
Index: linux/drivers/net/fjes/fjes_main.c
===================================================================
--- linux.orig/drivers/net/fjes/fjes_main.c
+++ linux/drivers/net/fjes/fjes_main.c
@@ -36,7 +36,7 @@ static const struct acpi_device_id fjes_
 	{ACPI_MOTHERBOARD_RESOURCE_HID, 0},
 	{"", 0},
 };
-MODULE_DEVICE_TABLE(acpi, fjes_acpi_ids);
+/* MODULE_DEVICE_TABLE(acpi, fjes_acpi_ids); */
 
 static bool is_extended_socket_device(struct acpi_device *device)
 {
